### DISCUSSION TITLE
Feature Wishlist

### DISCUSSION THREAD

**Discussion by revmischa** [echobind/prisma-factory#7] [Ideas]
I'm coming from the Python world, having used SQLAlchemy and Django ORM with https://factoryboy.readthedocs.io and https://faker.readthedocs.io/en/master/ and they're lovely to work with.

Wishlist:
* Integration with [Fishery](https://www.npmjs.com/package/fishery) to generate realistic fake data
* Option to either create an object in memory (`.build()` in factoryboy) vs. insert into the DB
* [LazyFunction](https://factoryboy.readthedocs.io/en/stable/reference.html#lazyfunction) 
* [LazyAttributeSequence](https://factoryboy.readthedocs.io/en/stable/reference.html#lazyattributesequence)
* [Iterator](https://factoryboy.readthedocs.io/en/stable/reference.html#iterator)

**Comment by benbender** [comment]
I would vote for an abstract interface for generators as there are more (f.e. https://chancejs.com/) and preferences (and codebases) may differ.

**Comment by andrewmclagan** [comment]
`buildMany` and `createMany` would be super helpful
